src/ng/animate.js

```diff
@@ -8,20 +119,90 @@ var $animateMinErr = minErr('$animate');
  *
  * @description
  * Default implementation of $animate that doesn't perform any animations, instead just
- * synchronously performs DOM
- * updates and calls done() callbacks.
+ * synchronously performs DOM updates and resolved the returned runner promise.
  *
- * In order to enable animations the ngAnimate module has to be loaded.
+ * In order to enable animations the `ngAnimate` module has to be loaded.
  *
- * To see the functional implementation check out src/ngAnimate/animate.js
+ * To see the functional implementation check out `src/ngAnimate/animate.js`.
  */
-var $AnimateProvider = ['$provide', function($provide) {
+var $$CoreAnimateQueueProvider = [function() {
+  var postDigestQueue = new HashMap();
+  var postDigestElements = [];
+
+  this.$get = ['$$AnimateRunner', '$$jqLite', '$rootScope', '$$animateClassResolver',
+       function($$AnimateRunner,   $$jqLite,   $rootScope,   $$animateClassResolver) {
```

Comment:
`$$jqLite` doesn't seem to be used here. Either use it (for consistency with the `ngAnimate` counterpart), or remove the dependency.